modelon/impact/client/entities/case.py

```diff
@@ -43,6 +52,20 @@ def consistent(self) -> bool:
         false otherwise."""
         return self._consistent
 
+    @property
+    def started(self):
+        """Case execution start time."""
+        if not self._datetime_started:
+            logger.warning("This attribute is deprectated, use 'run_info' instead")
```

Comment:
I don't understand this. Why is this method deprecated only under certain conditions? Also why is this method added if it is directly deprecated?

Reply:
Ok, I see. I think this should be documented in the doc-string rather than being a warning.